sway/commands/bind.c

```diff
@@ -110,6 +110,10 @@ static struct cmd_results *cmd_bindsym_or_bindcode(int argc, char **argv,
 
 	binding->command = join_args(argv + 1, argc - 1);
 
+	if (config->reading) {
+		argv[0] = expand_var(argv[0]);
```

Comment:
Sorry, I still don't get it: why is there a special case for this one? Isn't it expanded anyway before the handler is executed?

Reply:
Oh, I see, thanks for the explanation.